src/utils/utils.ts

```
@@ -18,6 +19,27 @@ import { I80F48, ONE_I80F48 } from './fixednum';
 import MangoGroup from '../MangoGroup';
 import { HealthType } from '../MangoAccount';
 
+export interface Block {
+  blockhash: string;
+  lastValidBlockHeight: number;
+}
+
+export const tryGetLatestBlockhash = async (
+  connection: Connection | undefined,
+  commitment: Commitment = 'confirmed',
+) => {
+  if (!connection) {
+    return;
+  }
+  try {
+    const block = await connection?.getLatestBlockhash(commitment);
+    return block;
+  } catch (e) {
+    console.log('Error getting blockhash');
+    return;
```

The exception is ignored by the `catch` block. Should we include some aspect of it (e.g. the message) in the log?

---

added